tests/unit/test_job_result.py

```diff
@@ -33,6 +33,17 @@ def test_job_result_qprogram_works():
     assert isinstance(job_result.data, dict)
 
 
+def test_job_result_vqa_works():
+    """Test vqa results are returned as dicts"""
+
+    job_result = JobResult(
+        job_id=0,
+        http_response="eyJ2cWFfZGljdCI6IHsiX25hbWUiOiAiVlFBIiwgImFuc2F0eiI6IHsiX25hbWUiOiAiSGFyZHdhcmVFZmZpY2llbnRBbnNhdHoiLCAiX2NpcmN1aXQiOiB7Il90eXBlIjogIlF1YW50dW1DaXJjdWl0IiwgIl9nYXRlcyI6IFtdLCAiX2luaXRfc3RhdGUiOiBudWxsLCAibl9xdWJpdHMiOiA0fSwgIm5fcXViaXRzIjogNCwgImxheWVycyI6IDEsICJjb25uZWN0aXZpdHkiOiBbWzAsIDFdLCBbMSwgMl0sIFsyLCAzXV0sICJzdHJ1Y3R1cmUiOiAiZ3JvdXBlZCIsICJvbmVfZ2F0ZSI6ICJVMiIsICJ0d29fZ2F0ZSI6ICJDTk9UIn0sICJiYWNrZW5kIjogeyJfdHlwZSI6ICJRaWJvIiwgIl9jaXJjdWl0IjogbnVsbCwgImJhY2tlbmQiOiAibnVtcHkiLCAicGxhdGZvcm0iOiBudWxsfSwgImNvc3RfZnVuY3Rpb24iOiB7Il9uYW1lIjogIlRTUF9Db3N0RnVuY3Rpb24iLCAiaW5zdGFuY2UiOiB7Il9uYW1lIjogIlRTUF9JbnN0YW5jZSIsICJuX25vZGVzIjogMiwgInN0YXJ0IjogbnVsbCwgImxvb3AiOiB0cnVlLCAiX2Rpc3RhbmNlcyI6IFtbMC4wLCAwLjk2MzU3ODM4OTk0MjE5NzFdLCBbMC4zMTYzMTgyNzgxOTEzMjAzLCAwLjBdXX0sICJwYXJhbWV0ZXJzIjogW10sICJlbmNvZGluZyI6ICJvbmVfaG90IiwgImxhZ3JhbmdlX211bHRpcGxpZXIiOiAxMH0sICJpbnN0YW5jZSI6IHsiX25hbWUiOiAiVFNQX0luc3RhbmNlIiwgIm5fbm9kZXMiOiAyLCAic3RhcnQiOiBudWxsLCAibG9vcCI6IHRydWUsICJfZGlzdGFuY2VzIjogW1swLjAsIDAuOTYzNTc4Mzg5OTQyMTk3MV0sIFswLjMxNjMxODI3ODE5MTMyMDMsIDAuMF1dfSwgIm9wdGltaXplciI6IHsiX25hbWUiOiAiR3JhZGllbnREZXNjZW50In0sICJzYW1wbGVyIjogeyJfbmFtZSI6ICJTYW1wbGVyIiwgIl9jaXJjdWl0IjogbnVsbCwgIl9wYXJhbWV0ZXJzIjogW10sICJfcXVhbnR1bV9zdGF0ZSI6IG51bGwsICJfcHJvYmFiaWxpdHlfZGljdCI6IG51bGwsICJfcmVxdWlyZWRfcXViaXRzIjogbnVsbH0sICJuX3Nob3RzIjogMTAwMH0sICJpbml0X3BhcmFtcyI6IFswLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwLCAwXSwgIm9wdGltaXplcl9wYXJhbXMiOiB7Im1ldGhvZCI6ICJQb3dlbGwifX0=",
```

Comment:
What is this?

Reply:
Something like this would be much easier to read/maintain/debug:

```
    EXPECTED_VQA_RESULTS = {"vqa_dict": {"_name": "VQA", "ansatz": {"_name": "HardwareEfficientAnsatz", "_circuit": {"_type": "QuantumCircuit", "_gates": [], "_init_state": null, "n_qubits": 4}, "n_qubits": 4, "layers": 1, "connectivity": [[0, 1], [1, 2], [2, 3]], "structure": "grouped", "one_gate": "U2", "two_gate": "CNOT"}, "backend": {"_type": "Qibo", "_circuit": null, "backend": "numpy", "platform": null}, "cost_function": {"_name": "TSP_CostFunction", "instance": {"_name": "TSP_Instance", "n_nodes": 2, "start": null, "loop": true, "_distances": [[0.0, 0.9635783899421971], [0.3163182781913203, 0.0]]}, "parameters": [], "encoding": "one_hot", "lagrange_multiplier": 10}, "instance": {"_name": "TSP_Instance", "n_nodes": 2, "start": null, "loop": true, "_distances": [[0.0, 0.9635783899421971], [0.3163182781913203, 0.0]]}, "optimizer": {"_name": "GradientDescent"}, "sampler": {"_name": "Sampler", "_circuit": null, "_parameters": [], "_quantum_state": null, "_probability_dict": null, "_required_qubits": null}, "n_shots": 1000}, "init_params": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], "optimizer_params": {"method": "Powell"}}

    job_result = JobResult(
        job_id=0,
        http_response=base64.b64encode(EXPECTED_VQA_RESULTS),
        job_type="vqa",
    )
    assert isinstance(job_result.data, dict)
```

Reply:
As this pattern is repeated all over the file, no need to fix it in this PR

Reply:
that's a nice small change we can reppeat arround the project. I'll make a ticket

Reply:
https://linear.app/qilimanjaro/issue/CLO-348/avoid-qiboconnection-base64-test-strings